mimic/rest/loadbalancer_api.py

```diff
@@ -18,6 +18,7 @@
 from random import randrange
 
 from mimic.util.helper import invalid_resource, json_dump
+from characteristic import attributes
```

Comment:
It's more compact only for the most naive use-cases.  If I want to use any validators, holy cow does it get wordy.  Very, very wordy.  :(